static/app/components/replaysOnboarding/utils.tsx

```diff
@@ -4,7 +4,12 @@ import {PlatformKey, replayPlatforms} from 'sentry/data/platformCategories';
 import {PlatformIntegration, Project} from 'sentry/types';
 
 export function generateDocKeys(platform: PlatformKey): string[] {
-  return ['1-install', '2-configure'].map(key => `${platform}-replay-onboarding-${key}`);
+  const platformKey = platform.startsWith('javascript')
+    ? platform
+    : 'javascript-' + platform;
```

Comment:
some platforms that allow replays aren't subplatforms of `javascript`, so we need to add this suffix so we can follow the same parsing format